qemu/tests/cfg/hotplug_virtio_mem.cfg

```diff
@@ -0,0 +1,32 @@
+- hotplug_virtio_mem:
+    only Linux
+    no RHEL.6 RHEL.7 RHEL.8
+    no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8
+    no s390x
+    type = hotplug_virtio_mem
+    virt_test_type = qemu
+    login_timeout = 240
+    threshold = 0.025
+    slots_mem = 20
+    maxmem_mem = 80G
+    mem_fixed = 4096
+    mem_devs = 'mem0 mem1'
+    guest_numa_nodes = 'node0 node1'
+    numa_memdev_node0 = mem-mem0
+    numa_memdev_node1 = mem-mem1
+    size_mem_mem0 = 2G
+    use_mem_mem0 = no
+    size_mem_mem1 = 2G
+    use_mem_mem1 = no
+    target_mems = "vmem0"
+    vm_memdev_model_vmem0 = "virtio-mem"
+    arm64-mmio:
+        virtio_mem_model = "virtio-mem-device"
```

Comment:
These 2 line is useless now, can be removed.

Reply:
Changed now, thanks!